tests/dismod/db/test_wrapper.py

```diff
@@ -189,42 +189,69 @@ def test_validate_data__missing_column():
     assert "nonnullable_column" in str(excinfo.value)
 
 
-def test_two_files_different_columns():
```

Comment:
This test should have had a docstring. I wrote it. It's the only test that creates two DismodFile objects with different added columns, and we should still be able to do that with the code you've written, and still test that.

Reply:
Hmm. Good point. I don't quite cover this case in my other tests. I'll bring it back.